test/lib/middleware/assignActiveLicence.spec.js

```diff
@@ -0,0 +1,143 @@
+const logger = require('./../../../lib/logger');
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const nock = require('nock');
+const httpMocks = require('node-mocks-http');
+const { assignActiveLicence } = require('./../../../index');
+const { acquisitionCtxClient } = require('@financial-times/kat-client-proxies');
+const licenceContext = require('./../fixtures/licenceContext.js');
+
+describe.only('assignActiveLicence', () => {
+	let getContextStub;
+	let logMessageStub;
+	const logMessages = [];
+
+	before(() => {
+		logMessageStub = sinon.stub(logger, 'log').callsFake((...params) => {
+			logMessages.push(params);
+		});
+	});
+
+	afterEach(() => {
+		nock.cleanAll();
+		getContextStub.restore();
+
+	});
+
+	after(() => {
+		nock.cleanAll();
```

Comment:
Do we need to `cleanAll` here if we are already doing it in `afterEach`?

Reply:
nope